portfolio.html

```diff
@@ -0,0 +1,606 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <nav>
+        <div class="lower-bar">
+          <div class="hamburger">
+            <img
+            src="bar/Enabled.png"
+            class="allign"
+            alt="hamburger menu symbol"
+          />
+          </div>
+          <div class="close1">
+            <p class="close-menu">&times;</p>
+          </div>
+          
+        </div>
+        <div class="contacts-menu">
```

Comment:
- Please just as the second reviewer has said, I notice opening your mobile menu for the second time, causes it to crash and shows a plain colored page, as in the image below. Please try fixing it.

##### Your design (The second attempt to open the mobile menu)
![Screenshot from 2022-07-20 09-19-46](https://user-images.githubusercontent.com/58771507/179934412-c48d21e5-1eee-493a-b1e3-3637e446abbc.png)

**Note** Please, feel free to reach out to me on slack `@kanu mike chibundu`, so we fix a one-on-one call to look into it, if you find it difficult.